lir/config/base.py

```diff
@@ -70,6 +71,24 @@ def parse(
         raise NotImplementedError
 
 
+class OutputConfigParser(ConfigParser):
```

Comment:
Deze klasse is bijna identiek aan `GenericConfigParser`, behalve dat hij checkt op de output type. Die check zou je ook in `experiment_strategies.py` kunnen inbouwen, waar hij de klasse opvangt, en dan is deze klasse niet meer nodig.